### PR TITLE
Plugwise docs updated to match pull-request #28237

### PR DESCRIPTION
**Pull request by bouwew**
**Description:**

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#28237

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/plugwise.markdown [hunk 1]
```diff
@@ -4,7 +4,7 @@ description: "Plugwise Climate integration."
 logo: plugwise.png
 ha_category: Climate
 ha_iot_class: Local Polling
-ha_release: 0.98
+ha_release: 0.102
```
Comment (klaasnicolaas) [inline source/_integrations/plugwise.markdown]: Why should you change the release version? Please revert his

File: source/_integrations/plugwise.markdown [hunk 2]
```diff
@@ -25,6 +25,7 @@ climate:
   - platform: plugwise
     password: YOUR_SHORT_IP 
     host: YOUR_SMILE_LOCAL_IP
+    legacy_anna: true # required for a legacy Anna (firmware 1.8.x), not needed for a more recent Anna (firmware 3.1.x)
```
Comment (klaasnicolaas) [inline source/_integrations/plugwise.markdown]: We want only required config variables in the first example, you can put the explenation in the description for `legacy_anna`

**Review by klaasnicolaas — DISMISSED** [review]
This must be merged in the `next` branch, please rebase this PR

**Comment by bouwew** [comment]
Sorry, how do I rebase to the next branch? This is all very new to me.

File: source/_integrations/plugwise.markdown (hunk 2 above)
Comment (bouwew) [reply source/_integrations/plugwise.markdown]: Rebased and changes made.

File: source/_integrations/plugwise.markdown (hunk 1 above)
Comment (bouwew) [reply source/_integrations/plugwise.markdown]: Revert done

**Review by frenck — APPROVED** [review]

**Pull request merged**